index.js

```diff
@@ -3,15 +3,20 @@ var AlexaUtterances = require("alexa-utterances");
 var SSML = require("./to-ssml");
 var alexa = {};
 
-alexa.response = function() {
+alexa.response = function(hasSession) {
```

Comment:
I haven't actually written the code, give it a shot. Obviously backwards compat breaking is bad, maybe we can minimize that once we see what it looks like.